src/views/EnemyDetail/EnemyDetail.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+import { useHistory } from 'react-router';
+import EnemyDetail from '../../components/Enemy/Enemy';
+import { deleteEnemy, fetchEnemyId } from '../../services/Enemies';
+
+export default function Enemy(props) {
+  const [enemy, setEnemy] = useState([]);
```

Comment:
enemy will end up as an object. It would make sense to set the default state to an empty object `{}`